src/main/resources/db/migration/V5__create_delivery_employee_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE deliveryEmployee (
+	delEmpId int NOT NULL AUTO_INCREMENT,
+    `name` varchar(50),
+    salary decimal(7, 2),
+    currency char(3),
+	bankAccountNumber varchar(30),
+    nationalInsurance varchar(30),
+    PRIMARY KEY(delEmpId)
```

Comment:
fix the formatting on this please
How long is a national insurance number? Do you need 30?
Whats delEmpId

Reply:
fixed